src/components/Room/RoomDetailImg/index.tsx

```diff
@@ -0,0 +1,48 @@
+import useClickArrow from 'pages/Room/RoomDetailPage/hooks/useClickArrow';
+import styles from './RoomDetailImg.module.scss';
+
+interface ImgProps {
+  imgUrl: string[] | null;
+}
+
+function RoomDetailImg({ imgUrl }: ImgProps) {
+  const { imgIndx, clickRightArrow, clickLeftArrow } = useClickArrow();
+  return (
+    <div>
+      {imgUrl ? (
+        <div className={styles['img-slider']}>
+          <button className={styles['img-slider__img-arrow']} type="button" onClick={clickLeftArrow}>
+            <img src="https://static.koreatech.in/assets/ic-room/left-arrow.png" alt="이전 화살표" />
+          </button>
+          <div className={styles['img-slider__img']}>
+            <img src={imgUrl[imgIndx]} alt="방 사진" />
+          </div>
+          <button
+            className={styles['img-slider__img-arrow']}
+            type="button"
+            onClick={() => clickRightArrow(imgUrl?.length)}
+          >
+            <img
+              src="https://static.koreatech.in/assets/ic-room/right-arrow.png"
+              alt="이후 화살표"
+            />
+          </button>
+          <div className={styles['img-slider__indx']}>
+            {imgIndx + 1}
+            {' '}
+            /
+            {' '}
+            {imgUrl.length}
+          </div>
```

Comment:
간단하게 아래처럼 줄일 수 있을 것 같아요.
옛날에도 이런 리뷰를 드렸던 것 같은 기억이? ㅋㅋㅋ
```suggestion
          <div className={styles['img-slider__indx']}>
            {`${imgIndx + 1} / ${imgUrl.length}`}
          </div>
```

Reply:
수정했습니다. [6bee8d8](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/39/commits/6bee8d800081d8f6cbeaee232f92ec771ae7935f)